mat/triband.go

```diff
@@ -369,3 +457,50 @@ func (t *TriBandDense) Trace() float64 {
 	}
 	return tr
 }
+
+func copySymBandIntoTriBand(dst *TriBandDense, s SymBanded) {
+	n, k, upper := dst.TriBand()
+	ns, ks := s.SymBand()
+	if n != ns {
+		panic("mat: triangle size mismatch")
+	}
+	if k != ks {
+		panic("mat: triangle bandwidth mismatch")
+	}
+	t := dst.mat
+	sU, _ := untransposeExtract(s)
+	if sbd, ok := sU.(*SymBandDense); ok {
+		s := sbd.RawSymBand()
+		if upper {
+			if s.Uplo == blas.Upper {
+				for i := 0; i < n; i++ {
+					ilen := min(k+1, n-i)
+					copy(t.Data[i*t.Stride:i*t.Stride+ilen], s.Data[i*s.Stride:i*s.Stride+ilen])
+				}
+			} else {
+				// dst is upper triangular, s is in lower triangle.
+				// Commented out due to lack of test coverage.
+				// for i := 0; i < n; i++ {
+				// 	ilen := min(k+1, n-i)
+				// 	for j := 0; j < ilen; j++ {
+				// 		t.Data[i*t.Stride+j] = s.Data[(i+j)*s.Stride+k-j]
+				// 	}
+				// }
+				panic("not implemented")
```

Comment:
These panics certainly don't look nice but is it really worth the trouble of implementing them and covering with tests? `dst` will most likely always be lower triangular, that eliminates two panics. `s` comes from the outside but how likely is that something that looks like a `SymBandDense` will be stored in the lower triangle?

Reply:
Do you want to add a TODO that can be done when someone hits this? (and below)

Reply:
Do you mean adding a TODO comment, extending the panic string or creating a ticket?

Reply:
I was thinking just to add a TODO comment.

Reply:
Done.